backend/src/db/migrations/20230917152623_add_unknown_genre.ts

```diff
@@ -0,0 +1,23 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'genres';
+
+const GENRE_KEY_COLUMN_NAME = 'key';
+
+const UNKNOWN_GENRE = { key: 'unknown', name: 'Unknown' } as const;
+
+const up = async (knex: Knex): Promise<void> => {
+  return await knex(TABLE_NAME)
+    .where(GENRE_KEY_COLUMN_NAME, UNKNOWN_GENRE.key)
+    .then(async (rows) => {
+      if (!rows.length) {
+        await knex(TABLE_NAME).insert(UNKNOWN_GENRE);
+      }
+    });
```

Comment:
I believe we can try to do it this way
```suggestion
    .insert(UNKNOWN_GENRE)
    .onConlict(GENRE_KEY_COLUMN_NAME)
    .ignore();
```